Shared/DsaResources.h

```diff
@@ -27,6 +27,7 @@ class DsaResources : public QObject
 {
   Q_OBJECT
 
+  Q_PROPERTY(QString arcGISMapsSDKVersion READ arcGISMapsSDKVersion CONSTANT)
```

Comment:
Since technically this can be empty to begin with, it should have a NOTIFY and a corresponding signal along with it when it changes.

Reply:
removed CONSTANT. added NOTIFY and a signal for it.